main.go

```diff
@@ -154,6 +166,12 @@ func main() {
 		}
 	}
 
+	setupLog.Info("Starting cron handler")
+	cronHandler.Start()
+	defer func() {
+		setupLog.Info("Stopping cron handler")
+		<-cronHandler.Stop().Done()
+	}()
```

Comment:
The cron timer was never being started. I've fixed that here.
But we really should have some e2e tests for etcdbackupschedule.

Reply:
See https://github.com/improbable-eng/etcd-cluster-operator/issues/181